moveit_ros/planning/rdf_loader/src/rdf_loader.cpp

```diff
@@ -103,21 +103,20 @@ rdf_loader::RDFLoader::RDFLoader(const std::string& urdf_string, const std::stri
   moveit::tools::Profiler::ScopedStart prof_start;
   moveit::tools::Profiler::ScopedBlock prof_block("RDFLoader(string)");
 
-  urdf::Model* umodel = new urdf::Model();
```

Comment:
yay, that one actually fixes potential memory leaks from what I can see :bouquet: